Console release notes 2022-09-28 (#219)
neondatabase/website: content/release-notes/2022-09-28.md

---
label: 'Console'
---

### What's new

- Console: Added `read`, `update`, `delete` project access scopes for OAuth applications.

> - Console: Added possibility to select access scopes and projects on the OAuth consent page.

Reviewer: ```suggestion
- Console: Added the ability to select access scopes and projects on the OAuth consent page.
```

> - Console: Added `protected: boolean` flag into roles public API response.

Reviewer: ```suggestion
- Console: Added a `protected: boolean` flag to the roles public API response.
```

- Console: Changed region and platform ids from serial numbers to strings in the public API.

> - Console: Changed operations id from serial number to UUID type in the public API.

Reviewer: ```suggestion
- Console: Changed the operations id from a serial number to a UUID type in the public API.
```

> - Control Plane: Added internal mechanism for projects relocation between pageservers.

Reviewer: ```suggestion
- Control Plane: Added an internal mechanism for relocating projects between pageservers.
```

- Control Plane: Stopped sending metrics to Grafana Cloud.
- Control Plane: Removed usage of the obsolete `zenith_ctl` binary.

> - Control Plane: Split monolithic project object into project, branches and endpoints in the internal database schema.

Reviewer: ```suggestion
- Control Plane: Split he monolithic project object into project, branches, and endpoints in the internal database schema.
```

### Bug fixes

> - UI: Fixed failure after receiving the empty response in the SQL Editor.

Reviewer: ```suggestion
- UI: Fixed a failure that occurred after receiving an empty response in the SQL Editor.
```

> - UI: Fixed an issue when switching between schemas in Tables UI did not reset the table name.

Reviewer: ```suggestion
- UI: Fixed an issue to ensure that table names are reset when switching between schemas in the Tables UI.
```

> - Control Plane: Fixed passwordless proxy connection to just created project.

Reviewer: ```suggestion
- Control Plane: Fixed the passwordless proxy connection for the most recently created project.
```

Reviewer: Ho @ololobus How about:
"Fixed an issue that occurred when creating a new project using a passwordless proxy connection.

Author: Sounds good to me! Reworded a bit, though
